### ISSUE TITLE
Colorscheme dependent Syntax error

### ISSUE THREAD

**Issue by lgalke** [dhruvasagar/vim-dotoo#43]
Dear @dhruvasagar,

First of all: I really enjoy your plugin for the beauty of handling todo notes in vim (without trying to copy the org mode). However, as I cleaned up my vim setup due to recent Vim8 release and started once again with @tpope 's vividchalk as a colorscheme. I noticed the following Error lines upon loading a .dotoo file:

```
Error detected while processing function <SNR>85_ReadTodoKeywords[18]..<SNR>85_ExtendHighlightingGroup:                                                                   
line    6:                                                                                                                                                                
E416: missing equal sign: links to MoreMsg  ctermfg=22 guifg=#005f00 term=bold cterm=bold gui=bold                                                                        
E416: missing equal sign: links to MoreMsg  ctermfg=22 guifg=#005f00 term=bold cterm=bold gui=bold 
```

Apart from these lines, everything works as expected. Loading the same dotoo file with any other color scheme, does not invoke those errors.

Presumably relevant line in vividchalk.vim:
```
125 highlight MoreMsg       guifg=#00AA00 ctermfg=Green
[...]
128 highlight Question      none                                                                                                                                         
129 high link Question      MoreMsg
```
It would be great to see a fix, probably just a small issue of the ExtendHighlightGroup function?

Thanks in advance,
lgalke

**Comment by dhruvasagar** [comment]
Thanks for reporting this, i'll have a look.

**Comment by dareenzo** [comment]
Hi,

as stated above I also get this issue, though I'm not using vivid chalk, as seen bellow.

<img width="1061" alt="screen shot 2019-02-04 at 19 12 17" src="https://user-images.githubusercontent.com/6130991/52270854-ccb14880-294a-11e9-840e-df2d65adacb0.png">

**Comment by dhruvasagar** [comment]
This happens when you open the dotoo file ?

**Comment by dhruvasagar** [comment]
What colorscheme are you using @dareenzo

**Comment by dareenzo** [comment]
Yes this happens when I open dotoo and org files. I made org files be dotoo with autocmd

`autocmd vimrc BufEnter,BufNewFile,BufRead *.org setlocal filetype=dotoo`

I'm using the snow theme - https://github.com/nightsense/snow

**Comment by dareenzo** [comment]
when using themes like NeoSolarized or Tomorrow-Night those errors don't happen

**Comment by dhruvasagar** [comment]
Yes, I am using dhruvasagar/vim-railscasts-theme and it doesn't happen for me either, let me check it out this week
